.travis.yml

```diff
@@ -8,8 +8,8 @@ python:
 sudo: false
 
 install:
-  - pip install --quiet --requirement requirements.txt
-  - pip install --quiet pylint
+  - pip install -r requirements.txt
+  - pip install pylint
```

Comment:
why'd you get rid of the `--quiet`s? the progress bars are pretty useless in CI output

also, it's probably a good idea to only use long flags in scripts (`--requirement` instead of `-r`)